src/db/main.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+
+const sequelize = new Sequelize('postgres', 'postgres', 'test1234', {
```

Comment:
Consider using environment variables for database credentials and host to improve security and flexibility. 🔒